mimic/test/test_core.py

```diff
@@ -63,18 +63,14 @@ def test_from_plugin_includes_all_plugins(self):
             dns_plugin.dns,
             cinder_plugin.cinder
         ))
-        # all plugsin should be on the internal listing
+        # all plugins should be on the internal listing
         self.assertEqual(
             plugin_apis,
             set(core._uuid_to_api_internal.values()))
         # the external listing should still be empty
         self.assertEqual(
             set([]),
             set(core._uuid_to_api_external.values()))
-        self.assertEqual(
-            len(plugin_apis),
-            len(list(core.entries_for_tenant('any_tenant', {},
-                                             'http://mimic'))))
```

Comment:
The error was `15 != 16` (or something like) because the Cinder mimic plugin now has two catalog entries while that section of the test assumes that each plugin only has only one catalog entry.

Judging by the test docstring I'd say that this part of the test is probably unrelated.

```
         Using the :func:`MimicRoot.fromPlugin` creator for a
        :class:`MimicCore`, the nova and loadbalancer plugins are included.
```